Added expiration times for email links; added known issue (#52)
mulesoft/docs-access-management: modules/ROOT/pages/managing-your-account.adoc

== Join an Existing Organization

> If there is an existing Anypoint Platform organization that you want to join, ask the organization administrator for an invitation. When an administrator invites you to the organization, you receive an email with a link to accept the invitation. The link in one week.

Reviewer: ```suggestion
If there is an existing Anypoint Platform organization that you want to join, ask the organization administrator for an invitation. When the administrator responds with an email invitation, you must click the included link within one week.
```

Reviewer: I'm still not too keen on "an Anypoint Platform organization"....

* Admin: An account with the organization administrator role associated to a different organization.
* Owner: An organization owner account for a different organization.

> After clicking the link in the invitation, you can choose the account to use to join the organization. The link expires in one week.

Reviewer: Do both the link to accept and the ability to choose an account expire in one week, or just the link to accept? Clarify.
```suggestion
After clicking the link in the invitation, you have one week to choose the account to use to join the organization. 
```

[NOTE]
If the account you want to use is already associated with another master organization, an exclamation mark is displayed next to it. This means that additional steps are needed before migrating the account to a new organization.

[[create_new_account]]
== Create a New Account Within an Existing Organization

> You can create a new account within an existing organization only if you receive an email invitation to join the organization. The link expires in one week.

Reviewer: ```suggestion
You can create a new account within an existing organization only if you receive an email invitation with a link to join the organization. The link expires in one week.
```

== Upgrade Your Account

To upgrade your Anypoint Platform trial account to an enterprise license, contact your account representative or https://www.mulesoft.com/contact[the MuleSoft sales team].

== Known Issue: Account Creation

> When you attempt to create an account from an email invitation, an error appears, erroneously stating your credentials are invalid.

Reviewer: ```suggestion
When you attempt to create an account from an email invitation to join an organization, and you use a username that already exists (even in a different organization), an error appears, erroneously stating your credentials are invalid.
```

This error occurs when a username already exists, even if it is under a different organization.

> If the username already belongs to you, you can move the username from your previous organization to the organization that sent you the email invitation.

Reviewer: "If the username belongs to you in another organization, you can move the name to the organization that sent you the email invitation."

What if the username is already being used by someone else?

Author: They'd have to pick another. If I'm sarathecoubian in MuleSoft and I click a link to join Salesforce, I would encounter this issue and would need my username to be moved.
